deployment/DOCKER_MORE_CLOSELY.md

```diff
@@ -2,32 +2,8 @@
 
 ## Apple M1 Platform
 
-***Attention:** For using Docker commands in Apple M1 environments!*
-
-```bash
-# set env variable for your shell
-$ export DOCKER_DEFAULT_PLATFORM=linux/amd64
-```
-
-For even more informations, see [Docker More Closely](#docker-more-closely)
-
-### Docker Compose Override File For Apple M1 Platform
-
-For Docker compose `up` or `build` commands, you can use our Apple M1 override file that specifies the M1 platform:
-
-```bash
-# in main folder
-
-# for production
-$ docker compose -f docker-compose.yml -f docker-compose.apple-m1.override.yml up
-
-# for production testing Docker images from DockerHub
-$ docker compose -f docker-compose.ocelotsocial-branded.yml -f docker-compose.apple-m1.override.yml up
-
-# only once: init admin user and create indexes and contraints in Neo4j database
-$ docker compose exec backend /bin/sh -c "yarn prod:migrate init"
-```
+To get more information about the Apple M1 platform you find our documentation in our main code, [here](/CONTRIBUTING.md#docker-more-closely).
```

Comment:
can we not remove the whole file? Whats the benefit if it just referes to other files and holds no content

Reply:
I have left it here as a hint, because someone who works on a deployment will probably not have a look into the contributions readme to find how one can dive into the Docker layer. Or?

Reply:
I deleted it @ulfgebhardt